keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/ConstraintState.kt

```diff
@@ -16,7 +17,9 @@ data class ConstraintState(
   val judgedBy: String? = null,
   val judgedAt: Instant? = null,
   val comment: String? = null,
-  val attributes: ConstraintStateAttributes? = null
+  val attributes: ConstraintStateAttributes? = null,
+  @JsonIgnore
+  var uid: String? = null
```

Comment:
I had to expose the `uid` here because I needed something short that I could include in the callback ID and that was unique.

Reply:
should the type here be UID?

Reply:
Ah, good catch. I didn't notice that type existed. Will take a look.